core/src/js/components/main-nav/Nav.js

```diff
@@ -265,15 +270,21 @@ export default class Nav {
     if (focusOnFirst) {
       this.focusOn('first'); // Focus on the first top level link
     }
+    // alert others the mobile nav has opened
```

Comment:
```suggestion
    // Alert others the mobile nav has opened.
```

Reply:
I really don't like the sentence-like construction of comments. Can we agree to disagree on that comment style?

Reply:
This is a convention that has been hammered into me by multiple other projects. I don't see anything that would make this a requirement in this project. Checking a few of the most popular projects on npm.org, I see the following:

lodash: Uses sentence structure
express: Doesn't use sentence structure
react: Capital start but no period.
chalk: Mixed, no format.
request: Capital start but no period.
commander: Capital start but no period.
Moment: Uses sentence structure
debug: Uses capitals but no period.

How about we split the difference? Comments must start with a capital letter but no punctuation is required at the end? That seems to be the common pattern.